.bazelrc

```diff
@@ -11,3 +11,16 @@ build --host_force_python=PY2
 # Opt-in to upcoming breaking changes
 build --incompatible_use_python_toolchains
 common --incompatible_string_join_requires_strings
+
+###############################
+# Remote Build Execution support
+# Turn on these settings with
+#  --config=remote
```

Comment:
should we turn on remote for CI?

Reply:
it's already enabled on the buildkite build, that's why I found the issue to begin with. (but local:1 was the wrong solution)